pkg/builders/builders.go

```diff
@@ -1,14 +1,19 @@
 package builders
 
-// OwnerNameLabel is the label used to mark ownership over a given resources.
+import (
+	"github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	"github.com/operator-framework/operator-marketplace/pkg/apis/operators/v2"
+)
+
+// CscOwnerNameLabel is the label used to mark ownership over a given resources.
```

Comment:
Given the two sets of constants, the description should be more explicit. `CscOwnerNameLabel is the label used to indicate the name of the CatalogSourceConfig that owns this resources` and so on. Please fix all the docs.